src/main/java/by/antonsh/arrays/Ex9.java

```diff
@@ -0,0 +1,43 @@
+package by.antonsh.arrays;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+/*
+Пользователь должен указать с клавиатуры положительное число, а
+программа должна создать массив указанного размера из случайных целых
+чисел из [0;15] и вывести его на экран в строку. После этого программа должна
+определить и сообщить пользователю о том, сумма какой половины массива
+больше: левой или правой, либо сообщить, что эти суммы модулей равны. Еслипользователь введёт неподходящее число, то выдать соответствующее
+сообщение
+ */
+public class Ex9 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int length;
+        System.out.println("Введите целое положительное число.");
+        do {
+            while (!scanner.hasNextInt()) {
+                System.out.println("Введите положительное целое число!");
+                scanner.next();
+            }
+            length = scanner.nextInt();
+        } while (length <= 1);
```

Comment:
we can enter also length = 1, it doesn't make much sense, but it's possible

Reply:
Ok, (length < 1)